app/models/message.rb

```diff
@@ -1,4 +1,6 @@
 class Message < ApplicationRecord
   belongs_to :user
-  belongs_to :chatroom
+  belongs_to :conversation
+
+  validates_presence_of :content, :conversation_id, :user_id
```

Comment:
belong to rồi thì ko cần validate 2 cái id kia nữa

Reply:
à quên đấy, cái case hôm trước của Thịnh nhỉ